### PR TITLE
Books

### PR DESCRIPTION
**Pull request by cvega08**
##Summary 
On lifecycle method created I made request for books, also create the styles for book.

##Screen 
books
Shot

##Trello Card
https://trello.com/c/0PWMVj2M

### CODE REVIEW

File: vue/wolox-books/src/router.js [hunk 1]
```diff
@@ -1,20 +1,37 @@
...
+    {
+      path: '*',
+      redirect: routes.home
     }
   ]
 })
+
+router.beforeEach((to, from, next) => {
+  const token = getToken()
+  to.fullPath === routes.login || to.fullPath === routes.sign_up ? next() : !token ? next(routes.login) : next()
```
Comment (FrankIglesias) [inline vue/wolox-books/src/router.js]: missing return here?

File: vue/wolox-books/src/services/BookService.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import api from '../config/api'
+import { getToken } from '../services/LocalStorageService'
+
+(() => {
+  const token = getToken()
+  token &&
```
Comment (FrankIglesias) [inline vue/wolox-books/src/services/BookService.js]: use an if here

File: vue/wolox-books/src/services/BookService.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import api from '../config/api'
+import { getToken } from '../services/LocalStorageService'
+
+(() => {
```
Comment (FrankIglesias) [inline vue/wolox-books/src/services/BookService.js]: remove this

File: vue/wolox-books/src/services/BookService.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  const token = getToken()
+  token &&
+    api.setHeader('Authorization', token)
+})()
```
Comment (FrankIglesias) [inline vue/wolox-books/src/services/BookService.js]: and this

File: vue/wolox-books/src/views/Home.vue [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+  height: 100%;
+  justify-content: space-around;
+  max-height: 60px;;
+}
+
+.icon-container {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+};
```
Comment (FrankIglesias) [inline vue/wolox-books/src/views/Home.vue]: remove semi colon

File: vue/wolox-books/src/router.js (hunk 1 above)
Comment (cvega08) [reply vue/wolox-books/src/router.js]: image
for it I use next()

**Pull request merged**